db/migrate/20161219225847_init_schema.rb

```diff
@@ -89,7 +88,7 @@ def up
       t.integer  "role"
       t.string   "otp_secret_key",                limit: 255
       t.integer  "second_factor_attempts_count",              default: 0
-      t.string   "phone",                         limit: 255
+      t.string   "phone_plain",                   limit: 255
```

Comment:
although as I look through the migrations, I don't see we ever created a PR to drop that column. Can be a follow on.